### PR TITLE
Unnecessarily gendered language in a comment

### PR DESCRIPTION
**Pull request by MPParsley**
Solves #470

### CODE REVIEW

File: tests/src/Kernel/Entity/SelectionHandlerTest.php [hunk 1]
```diff
@@ -133,14 +133,14 @@ public function testSelectionHandler() {
...
-   * We need to verify that each user get the groups he own in the normal widget
+   * We verify that each user gets the groups it owns in the normal widget
    * and the other users group's in the other groups widget and vice versa.
    */
   public function testSelectionHandlerResults() {
     $user1_groups = $this->createGroups(2, $this->user1);
     $user2_groups = $this->createGroups(2, $this->user2);
 
-    // Checking that the user get the groups he mange.
+    // Check that the user gets the groups he manages.
```
Comment (mradcliffe) [inline tests/src/Kernel/Entity/SelectionHandlerTest.php]: There's a "he" here as well, @MPParsley.

**Review by mradcliffe — COMMENTED** [review]
Found another term.

File: tests/src/Kernel/Entity/SelectionHandlerTest.php (hunk 1 above)
Comment (MPParsley) [inline tests/src/Kernel/Entity/SelectionHandlerTest.php]:
```suggestion
    // Check that the user gets the groups it manages.
```

Comment (MPParsley) [reply tests/src/Kernel/Entity/SelectionHandlerTest.php]: Yeah, so I noticed, updated it.

Comment (MPParsley) [reply tests/src/Kernel/Entity/SelectionHandlerTest.php]: FYI, you can propose inline code suggestions with cmd-g, awesome feature :-)

**Review by mradcliffe — APPROVED** [review]
It's passing now. :+1:

**Comment by zerolab** [comment]
I would suggest replacing he/she with "they", rather than "it"

File: tests/src/Kernel/Entity/SelectionHandlerTest.php [hunk 2]
```diff
@@ -133,14 +133,14 @@ public function testSelectionHandler() {
...
-   * We need to verify that each user get the groups he own in the normal widget
+   * We verify that each user gets the groups it owns in the normal widget
    * and the other users group's in the other groups widget and vice versa.
    */
   public function testSelectionHandlerResults() {
     $user1_groups = $this->createGroups(2, $this->user1);
     $user2_groups = $this->createGroups(2, $this->user2);
 
-    // Checking that the user get the groups he mange.
+    // Check that the user gets the groups it manages.
```
Comment (MPParsley) [inline tests/src/Kernel/Entity/SelectionHandlerTest.php]:
```suggestion
    // Check that users get the groups they manage.
```

File: tests/src/Kernel/Access/OgGroupContentOperationAccessTest.php [hunk 3]
```diff
@@ -174,7 +174,7 @@ protected function setUp() {
     }
 
     // Create a 'blocked' user. This user is identical to the normal
-    // 'authenticated' member, except that she has the 'blocked' state.
+    // 'authenticated' member, except that it has the 'blocked' state.
```
Comment (amitaibu) [inline tests/src/Kernel/Access/OgGroupContentOperationAccessTest.php]:
```suggestion
    // 'authenticated' member, except that they have the 'blocked' state.
```

(also change below it => they)

**Review by pfrenssen — APPROVED** [review]

**Pull request merged**